packages/notification-service/src/firebase.ts

```diff
@@ -137,6 +140,18 @@ export function initializeDb() {
     }
   )
 
+  lastInviteBlockRef.on(
+    'value',
+    (snapshot) => {
+      const lastBlock = (snapshot && snapshot.val()) || 0
+      console.debug('Latest invite block updated: ', lastBlock)
+      lastInviteBlockNotified = lastBlock
+    },
+    (errorObject: any) => {
+      console.error('Latest invite block read failed:', errorObject.code)
+    }
+  )
+
```

Comment:
Not specific to this addition, but I wonder if we should wait for the value to be read once, before starting the polling loop.
Right now, is it possible that the polling loop starts with uninitialized values?

Reply:
It can start with uninitialized values but both the `handleInvites`, `handleTransferNotifications` and `handlePaymentRequests` make checks for this in their first few lines so no unexpected behaviour should happen